opengrok-indexer/src/main/java/org/opengrok/indexer/history/FileHistoryCache.java

```diff
@@ -245,13 +245,17 @@ static History readCache(File file) throws IOException {
      * @param cacheFile the file to store the history to
      */
     private void writeHistoryToFile(File dir, History history, File cacheFile) throws HistoryException {
+
+        LOGGER.log(Level.FINEST, "writing history entries to ''{0}'': {1}",
+                new Object[]{cacheFile, history.getRevisionList()});
```

Comment:
It seems a bit weird calling `history.getRevisionList()` which creates a new list for a `FINEST` log but is executed every time. Maybe using the supplier like `LOGGER.log(Level.FINEST, () -> ...)` would be better?

However, maybe I'm just overthinking this, the actual performance impact should be negligible.

Reply:
That's a good point. I wrapped the added `FINEST` logs inside `isLoggable()`.